eng/code-quality-reports/src/main/java/com/azure/tools/checkstyle/checks/ExternalDependencyExposedCheck.java

```diff
@@ -149,10 +147,12 @@ private void checkNoExternalDependencyExposed(DetailAST methodDefToken) {
     private Map<DetailAST, String> getInvalidParameterTypes(DetailAST parametersTypeToken) {
         final Map<DetailAST, String> invalidParameterTypesMap = new HashMap<>();
         for (DetailAST ast = parametersTypeToken.getFirstChild(); ast != null; ast = ast.getNextSibling()) {
+//            log(ast, "ast type (getInvalidParameterTypes) ==== " + ast.getText());
             if (ast.getType() == TokenTypes.PARAMETER_DEF) {
                 invalidParameterTypesMap.putAll(getInvalidTypeFromTypeArguments(ast.findFirstToken(TokenTypes.TYPE)));
             }
         }
+//        log(parametersTypeToken, "Invalid Type (parametersTypeToken) MAP size = " + invalidParameterTypesMap.size());
```

Comment:
Why do we need to keep this here commented?  isn't it better to remove it?

Reply:
good catch! Clean some but still have some left here. Will clean it.